packages/fxa-auth-server/config/index.ts

```diff
@@ -1458,11 +1458,9 @@ const conf = convict({
   },
   forcePasswordChange: {
     forcedEmailAddresses: {
-      doc:
-        'Force sign-in confirmation for email addresses matching this regex.',
+      doc: 'Force password change for email addresses matching this regex.',
       format: RegExp,
-      default: '^$', // default is no one
-
+      default: /^forcepwdchange/,
```

Comment:
I think the default should still be no one

Reply:
Don't have a strong pref for this either way, only benefit is saving some space on pm2 config and specifying in cloud ops.